generate-defaults.ts

```diff
@@ -17,22 +17,23 @@ var TMP_DIR = process.env.EXT_TYPESCRIPT_DECLARATIONS_CACHE || '/tmp/typescript-
         { name: 'ExtJS-4.2.0.663', folder: 'ext-4.2.0.663', url: 'http://cdn.sencha.com/ext/gpl/ext-4.2.0-gpl.zip', jsduck_extra: ''},
         { name: 'ExtJS-4.2.1.883', folder: 'ext-4.2.1.883', url: 'http://cdn.sencha.com/ext/gpl/ext-4.2.1-gpl.zip', jsduck_extra: '' },
         { name: 'ExtJS-5.0.1',     folder: 'ext-5.0.1',     url: 'http://cdn.sencha.com/ext/gpl/ext-5.0.1-gpl.zip', jsduck_extra: 'ext-5.0.1/packages/sencha-*' },
-        { name: 'ExtJS-5.1.0',     folder: 'ext-5.1.0',     url: 'http://cdn.sencha.com/ext/gpl/ext-5.1.0-gpl.zip', jsduck_extra: 'ext-5.1.0/packages/sencha-*' }
+        { name: 'ExtJS-5.1.0',     folder: 'ext-5.1.0',     url: 'http://cdn.sencha.com/ext/gpl/ext-5.1.0-gpl.zip', jsduck_extra: 'ext-5.1.0/packages/sencha-*' },
+        { name: 'ExtJS-6.0.1',     folder: 'ext-6.0.1',     url: 'http://cdn.sencha.com/ext/gpl/ext-6.0.1-gpl.zip', jsduck_extra: 'ext-6.0.1/packages/sencha-* ext-6.0.1/packages/core*', theme : 'classic' }
```

Comment:
This is preventing Typescript compiling the file, on my system (with both Typescript 1.4 and 1.8):

```
gareth@gareth-N56VM:~/projects/typescript-declarations-for-ext$ npm install

> typescript-declarations-for-ext@0.0.1 prepublish /home/gareth/projects/typescript-declarations-for-ext
> tsc --module commonjs generate-defaults.ts && node generate-defaults.js

generate-defaults.ts(36,27): error TS2339: Property 'theme' does not exist on type '{ name: string; folder: string; url: string; jsduck_extra: string; } | { name: string; folder: st...'.
generate-defaults.ts(36,43): error TS2339: Property 'theme' does not exist on type '{ name: string; folder: string; url: string; jsduck_extra: string; } | { name: string; folder: st...'.
generate-defaults.ts(36,65): error TS2339: Property 'theme' does not exist on type '{ name: string; folder: string; url: string; jsduck_extra: string; } | { name: string; folder: st...'.
```